libnd4j/include/graph/impl/GraphUtils.cpp

```diff
@@ -18,7 +18,7 @@
 // Created by GS <sgazeos@gmail.com> 3/7/2018
 //
 
-#include "config.h"
+// #include "config.h"
```

Comment:
Here too, please don't comment those out.

Reply:
Don't worry, it's temporary comment.